asapdiscovery/data/fragalysis.py

```diff
@@ -66,6 +66,8 @@ def parse_xtal(x_fn, x_dir):
     x_dir : str
         Path to directory containing directories with crystal structure PDB
         files
+    p_only : bool, default=True
+        Whether to filter to only -P* files in fragalysis
```

Comment:
a quick explanation of what -P* files are would be helpful here

Reply:
new comment in [d36aa85](https://github.com/choderalab/covid-moonshot-ml/pull/101/commits/d36aa85956f6936d00eff149accf7da0e8bac400) that I think should help